jasa/paper-jasa.tex

```diff
@@ -47,7 +47,7 @@
 \if1\blind
 {
   \title{\bf Approximating Likelihood Ratios with Calibrated Discriminative Classifiers}
-  \author{Kyle Cranmer and Gilles Louppe\\
+  \author{Kyle Cranmer, Juan Pavez and Gilles Louppe\\

```

Comment:
I am currently preparing a notebook implementing generalized likelihood ratio tests based on a single parameterized classifier. I'll let you know as soon as I have something on which we can iterate together :) 

For the diagnosis, yes I meant using the performance of a classifier trained on `p0` versus reweighted `p1` as a proxy for evaluating the goodness of the estimated ratios.